be/src/runtime/snapshot_loader.cpp

```diff
@@ -203,12 +199,11 @@ Status SnapshotLoader::upload(const std::map<std::string, std::string>& src_to_d
                 }
             }
             ASSIGN_OR_RETURN(auto input_file, FileSystem::Default()->new_sequential_file(local_file_path));
-            auto res = fs::copy(input_file.get(), remote_writable_file.get(), 1024 * 1024);
-            if (!res.ok()) {
-                return res.status();
-            }
-            LOG(INFO) << "finished to write the file: " << local_file_path << ", length: " << *res;
+            ASSIGN_OR_RETURN(auto file_size,
+                             fs::copy(input_file.get(), remote_writable_file.get(), config::upload_buffer_size));
```

Comment:
will the `input_file` be closed automatically?

Reply:
yes, it does not provide close interface and will close automatically